0000-template.md

```diff
@@ -0,0 +1,76 @@
+- Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: [tauri-apps/rfcs#0000](https://github.com/tauri-apps/rfcs/pull/0000)
+- Discussion: [tauri-apps/rfcs#0000](https://github.com/tauri-apps/rfcs/discussions/0000)
```

Comment:
Should this be changed to "tracking issue" where after a PR is accepted/merged additional conversations/impls happen there?

Reply:
@wusyong suggested making the tracking mechanism more flexible. So instead of only being a tracking issue like I suggested also allowing an implementation PR to track status (for example if a PR is put in prior to an RFC being created).

Reply:
I think we can limit it to a tracking issue since any implementation specific PRs and other issues will be linked from that. No need to bloat the RFC template.

Reply:
should be good now?

Reply:
His point was what if you have impl PR first and then decide you need to actually do an RFC before merging. Then in that case you wouldn't necessarily need a tracking issue. But I think as long as we're flexible with whatever as this is our first pass let's go with as-is so we can get something useable